docs/user-guide/work-packages/work-package-table-configuration/README.md

```diff
@@ -123,6 +127,28 @@ To switch between the different criteria, open the [work package table configura
 
 ![display-settings](1566397517070.png)
 
+When choosing grouping the work package list by an attribute or by project a button to collapse groups shows up:
```

Comment:
Change "When choosing grouping the work package list by an attribute or by project a button to collapse groups shows up" to e.g. "When grouping the work package list by an attribute or by a project a button to collapse groups shows up"

Reply:
I think "When you group..." could be even better

Reply:
That's true. It's also an easier way of communicating.